tests/integration/test_actor_key_value_store.py

```diff
@@ -210,10 +211,16 @@ async def main() -> None:
             default_store_id = Actor.config.default_key_value_store_id
 
             store = await Actor.open_key_value_store()
-            record_url = await cast(KeyValueStoreClient, store._resource_client).get_public_url('dummy')
-            print(record_url)
-
-            assert record_url == f'{public_api_url}/v2/key-value-stores/{default_store_id}/records/dummy'
+            record_key = 'dummy'
+            record_url = await cast(KeyValueStoreClient, store._resource_client).get_public_url(record_key)
```

Comment:
Huh, I don't think we need to access `KeyValueStoreClient._resource_client` anymore - just `KeyValueStore.get_public_url` should be enough.